common/python/__init__.py

```diff
@@ -0,0 +1,3 @@
+#Automatically created by SCRAM
```

Comment:
Can you remove this file please? It's autogenerated by scram, and having it in may cause breakages later